db/tests/lookup_list_tests.rs

```diff
@@ -0,0 +1,52 @@
+use db_rs::Db;
+use db_rs::{Config, LookupSet};
+use db_rs_derive::Schema;
+use std::collections::HashSet;
+use std::fs;
+
+#[derive(Schema)]
+pub struct LookupSchema {
+    table1: LookupSet<u8, String>,
+    table2: LookupSet<u8, String>,
+    table3: LookupSet<u8, String>,
+    table4: LookupSet<u8, String>,
+    table5: LookupSet<u8, String>,
+}
+
+#[test]
+fn test() {
+    let dir = "/tmp/a/";
```

Comment:
you don't want this to collide with other tests, you could generate a random number or you can just see what the last letter used so far is and increment it